pkg/server/container_start.go

```diff
@@ -119,6 +138,12 @@ func (c *criService) StartContainer(ctx context.Context, r *runtime.StartContain
 		return nil, errors.Wrap(err, "failed to wait for containerd task")
 	}
 
+	if nric != nil {
+		if _, err := nric.InvokeWithSandbox(ctx, task, v1.Create, nriSB); err != nil {
```

Comment:
do you see an equivalence between sandbox and container here.. the naming is confusing given the concept of podsandbox has a different lifecycle to containers in the pod.

Reply:
talking with others, they want to make sure that both, the life cycle of sandboxes and containers are handled.

Reply:
fair.. and I definitely agree we need to manage both via. NRI.  We normally abstract away the concept of the pause container of the pod-sandbox as an internal implementation detail. CRUD events for pod would be at a higher level and include different config than it's sandbox pause container lifecycle events. I think we can do both and make sense of it, while allowing for some easy way to ignore pause container events.

Reply:
when we look at expanding out the sandbox from a simple pause container into a VM, this becomes more important.

Reply:
ok.. let's experiment away..